.github/workflows/iriscast_package_build.yaml

```diff
@@ -84,6 +84,8 @@ jobs:
       - name: Build RPM
         run: |
           cd iriscasttools
+          python_version=$(python -c "import sys; print(f'{sys.version_info.major}.{sys.version_info.minor}')")
```

Comment:
Shouldn't we use the matrix / var input from the CI rather than rely on Python echoing to us? Feels a bit like we're going to long way around

Reply:
We have python '3.x' in matrix so we can't directly get the python version

Reply:
In that case is it worth adding something like:
```
if: matrix.version != 3.x
```

Since we probably want to build wheels for specifically targetted major versions (hence 3.6 and 3.10)